modules/distr-tracing-architecture.adoc

```diff
@@ -25,6 +25,21 @@ This module included in the following assemblies:
 
 ** *Jaeger Console* – With the {JaegerName} user interface, you can visualize your distributed tracing data. On the Search page, you can find traces and explore details of the spans that make up an individual trace.
 
+* *{TempoName}* - This component is based on the open source link:https://github.com/grafana/tempo/[Grafana Tempo project].
+
```

Comment:
Could you please clarify for each item:

`Gateway` there is one or (possibly) more than one?
`Distributor` there  is one or (possibly) more than one?
`Ingester` there is one or (possibly) more than one?
`Query Frontend` there is one or (possibly) more than one?
`Querier` there is one or (possibly) more than one?
`Compactor` there one or (possibly) more than one?

Reply:
It's not important at all. I just need to know whether to use `a` or `the`.